resume.md

```diff
@@ -1,49 +1,49 @@
-# FIRST LAST NAME
+# VICTOR HUA
 
-(123) 456 - 7890 | email@gmail.com
+(201) 620 - 0088 | vhua@princeton.edu
 
-www.website.com | github.com/myusername | linkedin.com/myusername
+github.com/spoiledhua | linkedin.com/in/victorhua98
 
 ## EDUCATION
-**University of College** | B.S. Degree ... *MONTH 20XX*
-* **Coursework**: Data Structures, Algorithms, Upper Division CS Elective, Math Course, Statistics, Data Science, ETC.
+**Princeton University** | B.S.E. in Computer Science, Certificates in Finance, Statistics and Machine Learning *May 2021*
+* **Coursework**: Algorithms and Data Structures, Functional Programming, Graph Theory, Machine Learning,
+ Computational Geometry, Economics and Computing, Deep Learning, Financial Investments
 
 ## SKILLS
-* **Languages**: Python, Java, C, SQL, ...
-* **Libraries/Frameworks**: Pandas, TensorFlow, PyTorch, ...
-* **Version Control**: Git, Bitbucket, Sourcetree, GithubDesktop
-* *...*
+* **Languages**: Java, Python, JavaScript, Go, PostgreSQL, LaTeX
+* **Libraries/Frameworks**: Pandas, TensorFlow, React
+* **Version Control**: Git, GithubDesktop
 
 ## EXPERIENCE
-**Software Engineer**: Company | City, State ... *Month 20XX - Present*
-* Implemented XYZ Microservice using TECHNOLOGY to delivery PRODUCT to 123 Customers Globally
-* Architected CORE INFRASTRUCTURE LIBRARY used across X teams to BLAH BLAH
-* I did X for Y to accomplish Z
-* ...
+**Incoming Software Engineering Intern**: Optiver | Chicago, IL *Jun 2021 - Aug 2021*
```

Comment:
fine to put incoming - although if you do know the projects you'll be working on that can be helpful